lifecycle-operator/controllers/lifecycle/keptntask/job_utils.go

```diff
@@ -90,7 +95,8 @@ func (r *KeptnTaskReconciler) generateJob(ctx context.Context, task *klcv1alpha3
 					Annotations: task.Annotations,
 				},
 				Spec: corev1.PodSpec{
-					RestartPolicy: "OnFailure",
+					RestartPolicy:      "OnFailure",
+					ServiceAccountName: serviceAccountName,
```

Comment:
@bacherfl @thisthat @mowies @odubajDT should we also add the possibility to automount tokens? `AutomountServiceAccountToken`

Reply:
yes that makes sense since users can adjust the SA now

Reply:
@prakrit55 I would say yes, this should also be possible for users. Can you add it like you did for service account?

Reply:
got it, was waiting for the confirmation 🙌